internal/cluster/cluster.go

```diff
@@ -149,6 +149,9 @@ type Config struct {
 	MonitorBatchSize    int           `envconfig:"CLUSTER_MONITOR_BATCH_SIZE" default:"100"`
 }
 
+// Directory containing pre-generated TLS certs/keys for the ephemeral installer
+var clusterTLSCertOverrideDir string = `envconfig:"EPHEMERAL_INSTALLER_CLUSTER_TLS_CERTS_OVERRIDE_DIR" default:""`
```

Comment:
Shouldn't it be part of the Config? https://github.com/openshift/assisted-service/pull/7391/files#diff-ecbdbfaa3e0752590ccb7b5c0387c308de661ba622120129edf662c02fa1ddadR145
Or, just pass it directly from: https://github.com/openshift/assisted-service/pull/7391/files#diff-747a429a0e12307ef65d754c83b6898ce3858b621005e1e7af6e420da1f1657cR4068
I.e.
```
clusterPkg.CanDownloadKubeconfigNoIngress(cluster, b.Config.ClusterTLSCertOverrideDir)
```

Reply:
It does work as just a var but it is cleaner to only define once and pass as part of the bminventory. Updated.